Userland/Libraries/LibCodeComprehension/Cpp/Tests.cpp

```diff
@@ -89,9 +89,8 @@ int run_tests()
 
 static void add_file(FileDB& filedb, DeprecatedString const& name)
 {
-    auto file = Core::DeprecatedFile::open(LexicalPath::join(TESTS_ROOT_DIR, name).string(), Core::OpenMode::ReadOnly);
-    VERIFY(!file.is_error());
-    filedb.add(name, DeprecatedString::copy(file.value()->read_all()));
+    auto file = MUST(Core::File::open(LexicalPath::join(TESTS_ROOT_DIR, name).string(), Core::File::OpenMode::Read));
+    filedb.add(name, DeprecatedString::copy(MUST(file->read_until_eof())));
```

Comment:
You should use `MUST` if there should be no chance that an operation fails.

Here, it is more appropriate that you use `.release_value_but_fixme_should_propagate_errors()`.

Reply:
As I already responded to Lucas, this is not a fixme, as it is unfixable: The only correct behavior is to abort the test.

But okay, I guess I'll add a fixme then :shrug:

Reply:
I missed both that discussion _and_ the fact that this was part of test cases. Why is this not in `Tests/`? :cry:

Anyway, using `MUST` here makes sense in the context of tests. Although the commit message should be updated to reflect either the usage of `MUST` or `release_value_but_fixme_should_propagate_errors` if you choose to leave it that way.

Reply:
How about we leave it as a release_value_but_fixme_should_propagate_errors, and in a *different* PR I'll move the file to the Tests folder and change it to a MUST?

Reply:
Sounds great! As long as the commit message reflects the changes in this PR though :^)

Reply:
:+1: There already was *some* description, I expanded upon it a bit.